source/install-desktops/enable-reverse-proxy.rst

```diff
@@ -0,0 +1,149 @@
+.. _install-desktops-enable-reverse-proxy:
+
+Enable Reverse Proxy
+====================
+
+The reverse proxy will proxy a request to any specified host and port through
+IP sockets. This is different than what is used for proxying to per-user NGINX
+processes through Unix domain sockets. This can be used to connect to Jupyter
+notebook servers, RStudio servers, VNC servers, and more... This is disabled by
+default as it can be a security risk if not properly setup using a good
+``host_regex``.
```

Comment:
We can include that explanation, although there is one here in the documenation:

https://osc.github.io/ood-documentation/develop/infrastructure/ood-portal-generator/configuration.html#ood-portal-generator-configuration-configure-reverse-proxy

But we don't bring up `node` and `rnode` after that summary, so including it there may confuse them.